qemu/tests/netperf_udp.py

```diff
@@ -151,7 +151,7 @@ def run(test, params, env):
         netperf_server.start()
         # Run netperf with message size defined in range.
         msg = "Detail result of netperf test with different packet size.\n"
-        while(m_size <= end_size):
+        for m_size in range(start_size, end_size, step):
```

Comment:
Your change does not correct when `step = 1`, as `end_size` is excluded in `range`

Reply:
updated.